ansible/roles/nginx/tasks/main.yml

```diff
@@ -54,6 +54,13 @@
       - install
       - nginx
 
+  - name: Copy Lua Auth Script file
```

Comment:
Added task to remove the auth_request.lua when nginx_auth_enable is not defined or not true.